debian/changelog

```diff
@@ -1,3 +1,9 @@
+daos (2.3.107-3) unstable; urgency=medium
+  [ Lei Huang ]
+  * Add libcapstone as a new prerequisite package
+
+ -- Lei Huang <lei.huang@intel.com>  Thu, 11 May 2023 1:00:00 -0600
```

Comment:
Since you have to push an update, you could update the date here to get it more accurate to when this will land.

Reply:
Updated. Thank you!